src/main/java/model/input/file/PlainTextFile.java

```diff
@@ -76,29 +77,45 @@ private void skipToStartLine(Scanner scanner) throws IOException {
 	}
 
 	/**
-	 * Reads the lines from a scanner and inserts rows into the table.
+	 * Reads the lines from a scanner and inserts them into a list
 	 * @param scanner The scanner
+	 * @return The list containing all lines read from the start line
 	 */
-	private void readLines(Scanner scanner) {
-		while (counter <= getEndLine() && scanner.hasNextLine()) {
+	private ArrayList readLines(Scanner scanner) {
+		ArrayList<String> result = new ArrayList<>();
+		while (scanner.hasNextLine()) {
 			String line = scanner.nextLine();
-			String[] sections = line.split(delimiter);
-			List<Class<? extends DataValue>> columns = getColumnList();
-			DataValue[] values = new DataValue[getColumns().size()];
-			for (int i = 0; i < getColumns().size(); i++) {
-				values[i] = toDataValue(sections[i].trim(), columns.get(i));
-			}
-			builder.createRow(values);
+			result.add(line);
+		}
+		return result;
+	}
 
-			counter++;
+	private DataValue[] createRow(String line) {
```

Comment:
createRow suggereert dat deze functie een row teruggeeft. Zou het eerder getValues/createValues of iets degelijks noemen, of de rij in deze functie al toevoegen